configure.py

```diff
@@ -384,12 +384,16 @@ def set_action_env_var(environ_cp,
 def convert_version_to_int(version):
   """Convert a version number to a integer that can be used to compare.
 
+  Version strings of the form X.YZ and X.Y.Z-xxxxx are supported. The
```

Comment:
perhaps we could just stick a '1' onto the front before converting the digits to an int.  The numbers which are generated are never displayed so it wouldn't be an issue.